testsuite/python/test_checkpoint.py

```diff
@@ -792,6 +792,24 @@ def test_virtual_sites(self):
         np.testing.assert_allclose(
             np.copy(p_real.vs_relative[2]), [1., 0., 0., 0.], atol=1e-10)
 
+    @utx.skipIfMissingFeatures(['THERMAL_STONER_WOHLFARTH', 'EXTERNAL_FORCES'])
+    def test_thermal_stoner_wohlfarth_virtual_sites(self):
+        p_real, p_virt = system.part.by_ids([11, 12])
+        self.assertEqual(p_virt.magnetodynamics["is_enabled"],
+                         magnetodynamics_params.pop("is_enabled"))
+        for key, value in magnetodynamics_params.items():
+            np.testing.assert_allclose(p_virt.magnetodynamics[key], value)
+        Propagation = espressomd.propagation.Propagation
+        prop_flag = Propagation.TRANS_VS_RELATIVE | Propagation.ROT_VS_INDEPENDENT
+        self.assertEqual(p_real.propagation, Propagation.SYSTEM_DEFAULT)
+        self.assertEqual(p_virt.propagation, prop_flag)
+        self.assertEqual(p_real.vs_relative[0], -1)
+        self.assertEqual(p_virt.vs_relative[0], p_real.id)
+        self.assertEqual(p_real.vs_relative[1], 0.)
+        self.assertEqual(p_virt.vs_relative[1], 0.)
+        np.testing.assert_allclose(
+            np.copy(p_virt.vs_relative[2]), [1., 0., 0., 0.], atol=1e-10)
```

Comment:
Added a checkpointing test, seems to work as expected. But just for my understanding: in `Particle.hpp`, parameter `dt_incr` is not part of the `serialize()` function, and can be expected to be lost during ghost communication. Is this a quantity that gets regenerated from other parameters?